libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -371,12 +374,15 @@ public void startNavigation(NavigationViewOptions options) {
 
   private void setLocale(NavigationViewOptions options) {
```

Comment:
I think it would be good to refactor this method, as it's called `setLocale` but it's setting `Locale`, `unitType`, and `timeFormatType`.  For readability, it may be good to extract these each into their own private method.